jobs/sql/extractions/hiv_monthly_reporting.sql

```diff
@@ -30,7 +32,15 @@ dispensing_days_late			INT,
 latest_hiv_viral_load_obs_group	INT(11),
 latest_hiv_viral_load_date		DATETIME,
 latest_hiv_viral_load_coded		VARCHAR(255),
-latest_hiv_viral_load			INT
+latest_hiv_viral_load			INT,
+latest_arv_regimen_order_id		INT(11),
```

Comment:
Not sure what this means, since an ARV regimen could very well be represented by multiple orders